string type # text, html, img
string message # For text and html type
string file_path # For img type
uint8 img_size # For img type [percent]
